README.md

```diff
@@ -193,4 +193,4 @@ I am grateful to all previous project reviewers for their advice. Thanks to them
 
 This project is [MIT](./LICENSE) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please kindly ensure that when a category is created, users are redirected to the category homepage and not purchases index page. 👇 

<img width="358" alt="Untitled" src="https://github.com/umair-rao/Budget-app/assets/103386183/5e9bbbe8-bb74-4779-a187-073a2e06c381">

## Project Requirement 👇 

<img width="746" alt="Untitled" src="https://github.com/umair-rao/Budget-app/assets/103386183/e2198c5d-6917-4950-8e1d-b8c4ebc26289">

Reply:
Dear reviewer, It already navigates to transaction page you can all traction on the page (purchase) for which you have screenshot.

Reply:
- [ ] @umair-rao Great work implementing the changes requested, however as the previous reviewer pointed out, when a new category is created, the user should be redirected to the categories page in your case `http://127.0.0.1:3000/groups` but in your app am getting redirected to `http://127.0.0.1:3000/groups/1/purchases`, this is part of the requirement, the reviewer might have used the wrong illustration very sorry for that but her point is valid, this can be changed in the `groups controller` where you can change the path to redirect page on line 14 from ` redirect_to group_purchases_path(@group)` to `redirect_to groups_path`.
![image](https://github.com/umair-rao/Budget-app/assets/61408860/a77df43b-6bf7-471b-8a24-8933aad40b21)